src/mvp/objects/Animator.cpp

```diff
@@ -0,0 +1,26 @@
+#include "Animator.hpp"
+
+using namespace admirals;
+using namespace admirals::mvp::objects;
+
+int Animator::frameInd = 0;
```

Comment:
This can be moved to the header file by setting: `static inline int frameInd = 0;`